code/unclassified/src/addition_of_2_numbers_without_any_operator/addition.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+int add(int x, int y) 
+{
+    return printf("%*c%*c", x, '\r', y, '\r');
+}
+
+int main() 
+{
+    // test cases
+    int i, j;
+    for(i = 1; i <= 3; i++)
+    {
+        for(j = 1; j <= 3; j++)
+        {
+            printf("%d + %d = %d %c", i, j, add(i, j), '\n');
```

Comment:
Why not just directly print the newline?

Reply:
ohk i will :+1: